modules/tinymce/src/themes/silver/test/ts/browser/editor/bespoke/SilverBespokeButtonsTest.ts

```diff
@@ -92,11 +92,6 @@ describe('browser.tinymce.themes.silver.editor.bespoke.SilverBespokeButtonsTest'
     editor.off(eventName, eventTester);
   };
 
-  const assertNoEvent = () => {
```

Comment:
Event now always present.

Reply:
Instead of changing the test change the setContent setup so that it doesn't emit an event. I'm a bit uneasy to remove test assertions.

Reply:
But then we aren't testing the correct behavior, are we? If we start blocking events.

Reply:
Yes but the test are about moving on an initial state. So there is some setup, then some movement, then some assertions based on the movement.